src/anm/graphics/path.js

```diff
@@ -580,20 +541,38 @@ var encodeVisitor = function(segment, buffer) {
 Path.parse = function(path, target) {
     target = target || new Path();
     target.segs = [];
-    Path.visitStrPath(path, parserVisitor, target);
+    var segPaths = path.match(/[a-z][^a-z]*/ig);
```

Comment:
it may not be _way faster_, but the nice thing about it is that it works for any svg path according to the spec (previously, having two spaces or a comma would make the parsing fail).